Tita-Clone/Tita-Clone/Views/TextFieldView.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  TextField.swift
+//  Tita-Clone
+//
+//  Created by 혜인 on 2021/12/08.
+//
+
+import UIKit
+import SnapKit
+import Then
+
+class TextFieldView: UIView, UITextFieldDelegate{
+    lazy var textField = UITextField().then {
+        $0.backgroundColor = UIColor.white
+        $0.textColor = .black
+        $0.placeholder = "아이디"
+        $0.setPlaceholderColor(.rgb(red: 175, green: 175, blue: 175))
+        $0.dynamicFont(fontSize: 12, currentFontName: "NotoSans-Regular")
+        $0.delegate = self
+        $0.layer.cornerRadius = 10
```

Comment:
```suggestion
        $0.layer.cornerRadius = viewBounds.width/37.5
```
동적인 값을 주는 게 좋겠네요! 절대적인 값은 기종마다 다르게 보일 수 있습니다.